### DISCUSSION TITLE
[Airflow] error : invalid command 'db' bug 해결기

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#362] [1. 개발 관련 글 모음]
안녕하세요~ 
처음으로 discussions을 작성하네요.
이런 좋은 챌린지가 있다니, 그리고 참여할 기회를 주셔서 감사합니다.
덕분에 강제적(?)으로 꾸준히 공부를 해야할 명분이 생긴 것 같네요. 

제가 올릴 첫번째 글은 data pipeline 구성에 자주 사용되는 airflow라는 기술을 사용하면서 마주친 버그를 해결하는 과정이고 [blog](https://blog.naver.com/ds4ouj/223374267091)에 작성하였습니다.
앞으로 더 자주 글을 쓰도록 하겠습니다.

**Comment by shkisme** [comment]
우와 airflow.. 처음 보는 기술인데 이번 기회에 한번 살펴보도록 하겠습니다 👍👍

이미 블로그에 글이 엄청 많이 작성되어 있는데...ㄷㄷ!! 저도 더 열심히 글 써보도록 노력해야겠습니다!!! 🔥
